dotcom-rendering/src/web/components/CardHeadline.tsx

```diff
@@ -24,36 +25,50 @@ type Props = {
 	linkTo?: string; // If provided, the headline is wrapped in a link
 };
 
-const fontStyles = (size: SmallHeadlineSize) => {
+const fontStyles = (
+	size: SmallHeadlineSize,
+	containerPalette?: DCRContainerPalette,
+) => {
+	const options: FontScaleArgs = {};
+	if (containerPalette) options.fontWeight = 'bold';
```

Comment:
If this logic is just looking for container palette to be defined, would it be cleaner if this function took a parameter like `bold: boolean` and we did `fontSyles(size, !!containerPalette)` when calling it?

Reply:
Very tidy! Nice 👍